pkg/cmd/roachtest/roachtestutil/mixedversion/steps.go

```diff
@@ -715,3 +715,51 @@ func startStopOpts(opts ...option.StartStopOption) []option.StartStopOption {
 		option.NoBackupSchedule,
 	}, opts...)
 }
+
+type panicNodeStep struct {
+	rt         test.Test
+	initTarget int
+}
+
+func (s panicNodeStep) Background() shouldStop { return nil }
+
+func (s panicNodeStep) Description() string { return "run panic node" }
```

Comment:
For ease of debugging, lets have this say which node we are panicking. You'll have to select the node that's being restarted ahead of time, but I don't envision any issues with that.